app/models/trading_partner.rb

```diff
@@ -15,7 +15,7 @@ def validate_country
   end
 
   def self.options
-    GeographicalArea.all.map do |geographical_area|
+    GeographicalArea.all.sort_by(&:long_description).map do |geographical_area|
```

Comment:
This affects other parts of the site? Has this been cleared with UCD/Chris to change the ordering across the whole site?

Reply:
Moved this to CategoryAssessmentSearch